web-api/openapi/schemas/document.yml

```diff
@@ -61,6 +61,16 @@ DocumentProperty:
     - $ref: '#/DocumentPropertyArrayString'
     - $ref: './time.yml#/Timestamp'
 
+Snippet:
+  description: |-
+    The text of the found snippet.
+
+    This can be the whole ingested document or a part of it depending on how the
+    document from which it stems was ingested.
+
+    Be aware that whitespace can differ between the original and the returned snippet.
+  type: string
```

Comment:
This isn't quite the same type as `snippet` in `IngestedDocument` hence why I kept them apart.